host/sgx/enter.c

```diff
@@ -237,14 +228,44 @@ void oe_enter(
         if (code == OE_CODE_OCALL)
         {
             __oe_host_stack_bridge(
-                arg1, arg2, &arg1, &arg2, tcs, enclave, &ecall_context);
+                arg1, arg2, &arg1, &arg2, tcs, enclave, ecall_context);
         }
         else
             break;
     }
 
     *arg3 = arg1;
     *arg4 = arg2;
+
+    return OE_OK;
+}
+
+OE_NEVER_INLINE
```

Comment:
We may need to retain the current logic for Windows.  via #defines. The windows debugger may rely on a specific callstack.

Reply:
The only difference is an additional function call; i.e.,
`oe_enter (__morestack)`
versus
`oe_enter (__morestack)` -> `_enter_impl`

In the implementation-wise, original `oe_enter` should be identical to new `_enter_impl`.
Could the order of `oe_enter` affect Windows debugger contract (I assume `__morestack` aliasing is Linux/gdb-specific, but not sure)?